VoxelSniperFabric/build.gradle

```diff
@@ -0,0 +1,90 @@
+plugins {
+    id 'fabric-loom'
+    id 'maven-publish'
```

Comment:
Do we need the maven-publish plugin?

Reply:
I don't think we do actually. I'll remove it